packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -294,9 +294,18 @@ Widget _wrapWithBackground({
   );
 
   final bool darkBackground = backgroundColor.computeLuminance() < 0.179;
-  SystemChrome.setSystemUIOverlayStyle(
-    darkBackground ? SystemUiOverlayStyle.light : SystemUiOverlayStyle.dark
-  );
+  // TODO(jonahwilliams): remove once we have platform themes.
```

Comment:
in general i wouldn't bother with this kind of TODO. Either we eventually add platform themes and remove it, or we don't and this TODO isn't necessary.

I recommend reserving TODOs for more fundamental things where we know there's a limitation to the current implementation but we are intentionally cutting corners for some reason (e.g. because we are waiting an upstream fix).